models/marketplace.ts

```diff
@@ -70,7 +70,7 @@ export const ListingIO = t.intersection([
   ResourceURLIO,
   t.type({
     id: t.Integer,
-    status: makeEnumIOType(ListingStatusesEnum),
+    status: makeEnumIOType(InventoryStatusesEnum),
```

Comment:
I chose not to introduce this change yet, as this is only the case when users are authenticated. IMHO we should have 2 different codecs/types depending on whether the user is authenticated.

Reply:
Ah, I didn't know that difference, I only saw it not match.  Is it the user being authenticated at all, or the authenticated user being the owner of the listing?

I'll check out the code paths I encounter this and see what might make sense.